src/test/java/jeff/DeadlineTest.java

```diff
@@ -0,0 +1,42 @@
+package jeff;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.Test;
+
+public class DeadlineTest {
+    @Test
+    public void getSymbolTest() {
```

Comment:
your tests can be named `whatIsBeingTested_descriptionOfTestInputs_expectedOutcome` so that it is a bit clearer